packages/core/src/utils/graphFacade.ts

```diff
@@ -64,3 +66,8 @@ export class GraphFacade {
     return compact(nodes.map<IHttpOperation>(node => node.data as IHttpOperation));
   }
 }
+
+function transformLanguage(lang: string) {
```

Comment:
Suggest a simple map over function:

```ts
import { Languages } from '@stoplight/graphite/graph';

const langMap = {
  json: Languages.Json,
  yaml: Languages.Yaml,
  yml: Languages.Yaml,
}
```

This way can also check that `language` exists in `langMap` and report error if it does not.

```ts
language = langMap[language];
if (!language) {
  // report error and stop or whatever
}
```

Reply:
Ah interesting idea, I didn't think of reusing that part of graphite!

Reply:
@marbemac how about importing the function that does that in graphite and use it?

`import { filenameToLanguage } from '@stoplight/graphite/backends/filesystem/utils'`

…
`language: filenameToLanguage(path)`

Reply:
Even better! Still suggest reporting an error if language is undefined.

Reply:
The language will never be undefined; the function is using a proxied map returning the string you set in case it's not in the map 

https://github.com/stoplightio/graphite/blob/master/src/backends/filesystem/utils.ts#L15-L26

Reply:
Ah, got it - ok well then 🤷‍♂, hopefully folks don't feed prism files that are not json or yaml.